### DISCUSSION TITLE
I am taking this error. How can I fix this on my server.

### DISCUSSION THREAD

**Discussion by ghost** [SmartlyDressedGames/Unturned-3.x-Community#2861] [Q&A] [answered]
![Ekran Alıntısı](https://user-images.githubusercontent.com/88944657/137945763-c0972bb4-96d8-4571-b7ec-7e8c5362b151.JPG)

**Comment by ghost** [comment]
I am taking this error. How can I fix?

**Comment by SDGNelson** [comment, marked as answer]
More details here: https://github.com/SmartlyDressedGames/Unturned-3.x-Community/issues/2856#issuecomment-946102003

**Reply by ghost** [reply]
Okay so thanks. Nelson. And please add to Unturned RTX.
